### PR TITLE
Reformat has_nans

### PR DESCRIPTION
**Pull request by ohmdelta**
#1886

### CODE REVIEW

**Comment by ohmdelta** [comment]
@RitujaPawas could you review?

**Comment by RitujaPawas** [comment]
@ohmdelta Sorry for the delay. Can you please pull the latest changes? Thanks!

**Comment by RitujaPawas** [comment]
Also, please refer to : [docstrings](https://lets-unify.ai/ivy/deep_dive/10_docstrings.html)
and 
[docstring-examples](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html)

for adding for examples.

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -1256,13 +1256,55 @@ def has_nans(x: Union[ivy.Array, ivy.NativeArray], include_infs: bool = True) ->
...
+    >>> print(y)
+    False
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), b=ivy.array([3., 4., 5.]))
+    >>> y = ivy.has_nans(x)
+    >>> print(y)
+    False
+
```
[the remark this answers is not in the document]
Comment (ohmdelta) [reply ivy/functional/ivy/general.py:1396]: I pushed the changes for array and container and static methods with examples.

**Comment by ohmdelta** [comment]
Pushed changes for the files for `ivy/Array/general.py` and `ivy/Container/general.py`

**Comment by RitujaPawas** [comment]
@ohmdelta  make sure you request review everytime you have made changes according to previously made requests. Otherwise it slips from the radar.

**Comment by ohmdelta** [comment]
@RitujaPawas I've pulled. Could you review

File: ivy/container/general.py [hunk 2]
```diff
@@ -231,6 +232,121 @@ def all_equal(
...
+        ret
+            container of booleans, whether there is a nans at indices.
+
+        Examples
+        --------
+        With :code:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([1, 2, 3]), b=ivy.array([1, 2, float('nan')]))
+        >>> y = x.static_has_nans()
+        >>> print(y)
```
Comment (RitujaPawas) [inline ivy/container/general.py:277]: Please re-check examples implementations on static_has_nans()

File: ivy/functional/ivy/general.py [hunk 3]
```diff
@@ -1407,13 +1407,55 @@ def has_nans(x: Union[ivy.Array, ivy.NativeArray], include_infs: bool = True) ->
...
+    >>> x = ivy.array([float('inf'), 2, 3])
+    >>> y = ivy.has_nans(x, False)
+    >>> print(y)
+    False
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), b=ivy.array([3., 4., 5.]))
+    >>> y = ivy.has_nans(x)
+    >>> print(y)
```
Comment (RitujaPawas) [inline ivy/functional/ivy/general.py:1456]: Re-check the output in `ivy.Container` input specific example.

File: ivy/functional/ivy/general.py [hunk 4]
```diff
@@ -1407,13 +1407,55 @@ def has_nans(x: Union[ivy.Array, ivy.NativeArray], include_infs: bool = True) ->
...
+    >>> x = ivy.array([float('inf'), 2, 3])
+    >>> y = ivy.has_nans(x)
+    >>> print(y)
+    True
+
+    >>> x = ivy.array([float('inf'), 2, 3])
+    >>> y = ivy.has_nans(x, False)
+    >>> print(y)
+    False
+
```
Comment (RitujaPawas) [inline ivy/functional/ivy/general.py:1451]: Add examples using ivy.NativeArray, and Container instance method as well.

**Review by RitujaPawas — COMMENTED** [review]
Every thing looks good. Just a few examples to add! Request a review when you are done! 😀

**Comment by ohmdelta** [comment]
@RitujaPawas added and fixed the examples

**Comment by RitujaPawas** [comment]
Changes look good. I will take a final look and let you know! @ohmdelta

**Comment by RitujaPawas** [comment]
lgtm! Thankyou for your contribution! @ohmdelta

**Pull request merged**